app/components/add-proposal/component.js

```diff
@@ -13,47 +14,51 @@ export default Component.extend({
   kredits: service(),
 
   proposal: null,
+  contributors: null,
   inProgress: false,
 
   isValidRecipient: computed('proposal.recipientAddress', function() {
-    return this.get('kredits.web3Instance').isAddress(this.get('proposal.recipientAddress'));
+    return this.get('kredits.web3').isAddress(this.get('proposal.recipientAddress'));
   }),
 
   isValidAmount: computed('proposal.amount', function() {
-    // TODO
-    return true;
+    return parseInt(this.get('proposal.amount')) > 0;
```

Comment:
The second parameter of `parseInt` should always be `10` if you want it to be parsed as decimal number. So in this case it should be `parseInt(this.get('proposal.amount'), 10)`.
Otherwise, in certain situations the number will be interpreted with a different base (e.g. when the number starts with a 0, it will be octal).